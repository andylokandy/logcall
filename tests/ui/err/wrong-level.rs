#[logfn::logfn("foo")]
fn f() {}

fn main() {}
