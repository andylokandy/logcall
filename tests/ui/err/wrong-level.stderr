error: unknown log level
 --> tests/ui/err/wrong-level.rs:1:1
  |
1 | #[logfn::logfn("foo")]
  | ^^^^^^^^^^^^^^^^^^^^^^
  |
  = note: this error originates in the attribute macro `logfn::logfn` (in Nightly builds, run with -Z macro-backtrace for more info)
